dms/views/dms_portal_templates.xml

```diff
@@ -102,15 +100,12 @@
                                         t-att-title="dms_file.name"
                                     >
                                         <img
-                                            t-att-src="image_data_uri(dms_file.thumbnail)"
-                                            style="width:30px"
+                                            class="o_portal_contact_img"
+                                            t-att-src="dms_file.icon_url"
```

Comment:
No `t-if` here?

Reply:
No, I think that the icon is just enough for all cases in the portal. The thumbnail doesn't seem very useful there.